src/pkgcheck/checks/codingstyle.py

```diff
@@ -1035,3 +1035,42 @@ class EclassUnquotedVariablesCheck(_UnquotedVariablesCheck):
     def feed(self, eclass):
         for var_name, lines in self._feed(eclass):
             yield EclassUnquotedVariable(var_name, lines=lines, eclass=eclass.name)
+
+
+class ExcessiveLineLength(results.LinesResult, results.Style):
+    """Line is longer than 80 characters."""
```

Comment:
@thesamesam Low priority on your tasks, but when you have time, can you please suggest better class docs and `desc` (`{self.lines_str}` translates into `on lines [1,2]`)